packages/react-server/package.json

```diff
@@ -34,7 +34,7 @@
     "mobile-detect": "1.3.0",
     "q": "1.4.1",
     "querystring": "0.2.0",
-    "request-local-storage": "1.0.0",
+    "request-local-storage": "1.1.0",
```

Comment:
I think this part already landed in #617.  A rebase should get it out of the diff here.